app/views/career_site/positions/index.html.slim

```diff
@@ -1,5 +1,8 @@
-#positions.container.container-small
-  h1.mb-2 = t("career_site.positions.open_positions_header", name: @current_tenant.name)
+- content_for :title do
+  = t("career_site.positions.positions_title", company_name: @company_name)
+
+#positions
+  h1.mb-4 = t("career_site.positions.open_positions_header", name: @current_tenant.name)
```

Comment:
Nitpick: why for the same text we use different variables: above it's a @company_name, here it's a @current_tenant.name?
Also, probably good to have the same variable name for same text on all related locales, i.e. company_name instead of name